requirements_dev.txt

```diff
@@ -35,6 +35,9 @@ xgboost
 lightgbm
 rdkit-pypi; sys_platform != 'darwin' or (sys_platform == 'darwin' and platform.machine != 'arm64')
 rdkit-pypi; python_version > '3.7' and sys_platform == 'darwin' and platform.machine == 'arm64'
+httpx>=0.22.0  # 0.23.0 dropped Python 3.6; we can upgrade once we drop it too
```

Comment:
If I increase/remove this bound, then our py36 tests fail, because [our unit tests `pip install -r requirements_dev.txt`](https://github.com/wandb/wandb/blob/545e14483028a78afa4f57859be37f8fc501f867/tox.ini#L49) to get their dependencies; without this bound, py36 fails because it can't find a satisfactory httpx version.